pkg/config/config.go

```diff
@@ -105,10 +120,12 @@ func (c *Config) Set(keys []string, value string) {
 
 // Read gh configuration files from the local file system and
 // return a Config.
-func Read() (*Config, error) {
-	// TODO: Make global config singleton using sync.Once
-	// so as not to read from file every time.
-	return load(generalConfigFile(), hostsConfigFile())
+var Read = func() (*Config, error) {
```

Comment:
Changing this to be a variable for testing purposes seemed to be the best approach here to bypass having to deal with `sync.Once` in tests.